Dockerfile-Dev

```diff
@@ -1,5 +1,11 @@
-FROM node:12
-WORKDIR /code
-COPY package*.json /code/
-RUN npm install
-RUN npm install -g gulp-cli mocha
+FROM node:12
+
+# Install global packages
+RUN npm install -g gulp-cli mocha
```

Comment:
I had to add `@vue/cli-service` here in order to get this running. Otherwise the habitica_client container would error out saying it couldn't find `vue-cli-service`.

Otherwise the patch functions as intended.

Reply:
`npm install postinstall` was supposed to have installed `@vue/cli-service`. One thing to note, when running the command to start the containers, I've been running this:

`docker-compose -f docker-compose.dev.yml up --build`

Adding `--build` to the end of the command rebuilds the container.

Reply:
My apologies, you're absolutely correct. I thought I had cleaned up the old images before I rebuilt, but that was not the case. 

Patch works as expected.